### DISCUSSION TITLE
Store Authors in Audiobook folder

### DISCUSSION THREAD

**Discussion by netnft** [advplyr/audiobookshelf#2450] [Ideas]
Be neat to see the ability to save author information and images "with item" like you can with metadata and covers. The folder structure can support this no? I only ask because I cant for the life of me get the author info to stay after an update, but the "with item" function allows me to keep the rest. The ability to keep everything in the one library would be great.

**Comment by nichwall** [comment]
How are you running the server? If you're using Docker, did you set up a mount point for both `/config` and `/metadata`? If not, that's why you're losing data on updates.

**Reply by netnft** [reply]
Portainer on Qnap. Ive tried everything I know how to do but nothing saves to the folder mount points I set. I have been using the backup feature to keep most of the information.

      - /share/CACHEDEV4_DATA/Audiobooks:/Audiobooks:rw/
      - /share/CACHEDEV4_DATA/metadata:/metadata:rw/
      - /share/CACHEDEV4_DATA/config:/config:rw/
      
    this is what is set but it doesnt do anything.

**Comment by netnft** [comment]
Outside of the authors, everything else works fantastic.

**Comment by netnft** [comment]
Or at least the images of the authors. I just realized the metadata stays, as well as the list of authors and narrators, but the images don't. looking at it with more detail. Its a low priority thing but it would be cool to see. Since other data can be saved to these locations. Thanks!

![image](https://github.com/advplyr/audiobookshelf/assets/98012560/a2fef11c-57b0-4dee-b576-68226ffcbca8)

**Comment by Thomas-Magnum** [comment]
I've dozens of authors, where the automatic match function does not work properly - either the author is not found at all, the description is in an incorrect language or the image is not okay. 
I can copy the description via copy/past to audiobookshelf, but the option to link images only by URL is a pain. Workaround would be to allow for an upload of the image (like Plex does), however the best solution for me would be the way as it is proposed here:
Let audiobookshelf use the information provided in a text-file and separate JPG-file in the authors folder.
